server/util.go

```diff
@@ -271,12 +272,13 @@ func AddCommands(rootCmd *cobra.Command, defaultNodeHome string, appCreator type
 		ShowAddressCmd(),
 		VersionCmd(),
 	)
+
 	startCmd := StartCmd(appCreator, defaultNodeHome)
 	addStartFlags(startCmd)
 
 	rootCmd.AddCommand(
 		startCmd,
-		UnsafeResetAllCmd(),
+		tmcmd.ResetAllCmd,
```

Comment:
Just curious: did you ever get to test this cmd with the changes applied?

Reply:
I did! The command works, but panics lol. I think the TM is on it and will release a patch update soon.

Reply:
Actually, I'll fix it myself.

Reply:
Can you create an issue to remember to update it

Reply:
Nvm, just gonna wait till they get a patch out as it shouldn't take long :p